dalle2_pytorch/train_diffusion_prior.py

```diff
@@ -0,0 +1,160 @@
+import argparse
+import os
+from dalle2_pytorch import DiffusionPrior
+from embedding_reader import EmbeddingReader
```

Comment:
can you add that dependency in there https://github.com/lucidrains/DALLE2-pytorch/blob/main/setup.py#L24 ?

Reply:
@krish240574 last nitpick :)